app/javascript/containers/RestaurantContainer.js

```diff
@@ -28,11 +39,45 @@ class RestaurantContainer extends Component {
       .catch(error => console.error(`Error in fetch: ${error.message}`));
   }
 
+  postReview(review){
```

Comment:
Soooooo... because you have this fetch post... you need to test it, which means using `fetchMock` for POST (see bottom of that article)

Reply:
added fetchmock